### DISCUSSION TITLE
We moved JCB forum here ;)

### DISCUSSION THREAD

**Discussion by Llewellynvdm** [joomengine/Joomla-Component-Builder#621] [General]
Thanks to @Joomla I was made aware of this new discussion option on GitHub, and realized it would be ideal to bring these two worlds together.

So the discussion area is the new [forum of JCB](https://vdm.bz/jcb-forum), where we can communicate all things JCB, and that which is being build with JCB or if you need some kind of help in relation to your development of your components.

Then if you run into any issues, related to the JCB code, like bugs, start by searching the (open & closed) issues, and if the issue has not been mentioned before, then only open a new issue, following the [community guidelines](https://github.com/vdm-io/Joomla-Component-Builder/blob/master/.github/CONTRIBUTING.md).

**Comment by micker** [comment]
very cool thnaks for your component and support
